CA.md

```diff
@@ -0,0 +1,73 @@
+# Example CA Usage
+
+Although there is more than one way to handle extensions (grants/identities) and
+signing certificates, HIBA includes a `hiba-ca.sh` tool and `hiba-gen` binary.
+The following serves as a simple example of a way to get started with HIBA.
+
+Once you have completed the
+[compilation instructions](https://github.com/google/hiba/blob/main/INSTALL.md#compilation)
+you will have a `hiba-gen` and `hiba-ca.sh` in your hiba directory. You can now
+create CA keys, grants, identities, grant eligibility, and sign host and user
+certificates with these extensions. The following assumes you are in your hiba
+directory. The CA structure will be in `~/.hiba-ca` by default.
+
+1. Generate a CA keypair.
+    ```
+    $ ./hiba-ca.sh -c -- -N ""
```

Comment:
This is not needed, ssh-keygen will use an empty passphrase if the user presses [enter] at prompt time.

Since I suspect some people will be using these instructions not just for test, it would be better to give them the choice of using a secret passphrase or not, rather than making the call they do not need one.

Reply:
then you can also leave the `-- -N "" ` from all other commands, making them simpler. This -N is mostly needed when using in non-interactive more like in the tests.